.github/CODEOWNERS

```diff
@@ -5,8 +5,43 @@
 # the repo. Unless a later match takes precedence,
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
-*       @flacatus @jkopriva @rhopp @sawood14012 @psturc @albarbaro @kasemAlem @jinqi7 @tisutisu @srivickynesh @cuipinghuo @dheerajodha
+*       @flacatus @jkopriva @rhopp @sawood14012 @albarbaro @tisutisu @rsoaresd @psturc @kasemAlem @jinqi7 @srivickynesh @cuipinghuo @dheerajodha @naftalysh
+
+# Any change inside the `tests/build` `pkg/utils/jvmbuildservice` and `pkg/utils/build` directories will require approval from following members 
+tests/build @psturc 
+pkg/utils/build @psturc 
+pkg/utils/jvmbuildservice @psturc 
+
+# Any change inside the `tests/boyc` and `pkg/utils/gitops` directories will require approval from following members 
+tests/boyc @flacatus @jkopriva @rhopp @sawood14012 @albarbaro @tisutisu @rsoaresd
+pkg/utils/gitops @flacatus @jkopriva @rhopp @sawood14012 @albarbaro @tisutisu @rsoaresd
+
+# Any change inside the `tests/cluster-registration`  directories will require approval from following members 
+tests/cluster-registration @flacatus @jkopriva @rhopp @sawood14012 @albarbaro @tisutisu @rsoaresd 
+
+# Any change inside the `tests/enterprise-contract` and `pkg/utils/tekton`directories will require approval from following members 
+tests/enterprise-contract @psturc @cuipinghuo
+pkg/utils/tekton @psturc @cuipinghuo
+
+# Any change inside the `tests/integration-service` and `pkg/utils/integration` directories will require approval from following members 
```

Comment:
remove psturc if he is in global owner

Reply:
Are the Code Owners here restricted to be the QE engineers ? Can we also add few developers from the integration-service ?